map_model/src/edits/mod.rs

```diff
@@ -110,6 +112,9 @@ impl EditRoad {
             modal_filter: None,
             // TODO From crossing_nodes?
             crossings: Vec::new(),
+            // TODO - read these from osm tags. Where is this already parsed?
```

Comment:
In short: for this PR, keep things as they are. We should hash this out later and probably totally get rid of `get_orig_from_osm`.

The TODOs above for `modal_filter` and `crossings` reflect the same kind of problem. We don't currently store enough stuff on `Road` or elsewhere to derive the "original" value for TRs, crossings, or modal filters. The two TR fields are calculated in osm2streets, and copied here at https://github.com/a-b-street/abstreet/blob/1ad94007ece184195bb044310bbcc621e49841e3/map_model/src/make/mod.rs#L152, when we turn a RawMap (an osm2streets StreetNetwork object + buildings + some other stuff) into a Map.

It's been a while since I last thought through the map edit system, but I think my conclusion was that this `get_orig_from_osm` method is fundamentally flawed. It's kind of an accident that we can recreate lanes and speed limit from OSM tags. I think I concluded that the "original" state of something in map model shouldn't be "cached" or stored as a copy on `Road` or something else, and instead we should be able to figure it out from `MapEdits.original_roads`. Tracing through, `get_orig_from_osm` only has a few callers, and the relevant one here only happens when initially loading edits from a savefile.

Reply:
I have updated the comments but left the code unchanged.